Podfile

```diff
@@ -345,6 +345,17 @@ target 'WordPressStatsWidgets' do
 
   shared_with_all_pods
   shared_with_networking_pods
+  shared_with_extension_pods
+
+  wordpress_ui
+end
+
+target 'JetpackStatsWidgets' do
+  project 'WordPress/WordPress.xcodeproj'
+
+  shared_with_all_pods
+  shared_with_networking_pods
+  shared_with_extension_pods
```

Comment:
Yes, I am aware of these additional pod inclusions. They happen because I started using app-specific color for Jetpack and WordPress widgets.

I use `UIColor+MurielColor` that exposes `UIColor.primary` extension. It uses `MurielColor`, which uses `AppStyleGuide`, which depends on `GridIcons`. 

I can only include `GridIcons` so `ZIPFoundation` and `Down` wouldn't be included.